.github/workflows/reusable-build-test-release.yml

```diff
@@ -615,9 +615,27 @@ jobs:
           version: ${{ steps.BuildVersion.outputs.VERSION }}
       - name: Slim Package
         id: slim
-        uses: splunk/addonfactory-packaging-toolkit-action@v1
-        with:
-          source: ${{ steps.uccgen.outputs.OUTPUT }}
+        run: |
+          pip install semantic_version==2.6.0
+          pip install https://download.splunk.com/misc/packaging-toolkit/splunk-packaging-toolkit-1.0.1.tar.gz
+          INPUT_SOURCE=${{ steps.uccgen.outputs.OUTPUT }}
+          SOURCE_REGEX='^.*/$'
+          if [[ $INPUT_SOURCE =~ $SOURCE_REGEX ]];then
+            echo Removing trailing / from INPUT_SOURCE slim is picky
+            INPUT_SOURCE=$(echo $INPUT_SOURCE | sed 's/\(.*\)\//\1/')
+          fi
+          slim generate-manifest "${INPUT_SOURCE}" --update >/tmp/app.manifest   || true
+          cp  /tmp/app.manifest  "${INPUT_SOURCE}/app.manifest"
+          mkdir -p build/package/splunkbase
+          slim package -o build/package/splunkbase "${INPUT_SOURCE}" 
+          for f in build/package/splunkbase/*.tar.gz; do
+            n=$(echo "${f}" | awk '{gsub("-[0-9]+.[0-9]+.[0-9]+-[a-f0-9]+-?", "");print}' | sed 's/.tar.gz/.spl/')
+            mv "${f}" "${n}"
+          done
+          PACKAGE=$(ls build/package/splunkbase/*)
+          slim validate "${PACKAGE}"
+          chmod -R +r build
+          echo "OUTPUT=$PACKAGE" >> "$GITHUB_OUTPUT"
```

Comment:
If this `echo` has no effect, let's remove it

Reply:
but don't we need that echo for `upload-build-to-s3` to work correctly?

Reply:
it seems it needs it